src/schema/ecommerce/orders.ts

```diff
@@ -54,8 +66,10 @@ export const Orders = {
           last: $last
         ) {
           ${PageInfo}
```

Comment:
we should move this `totalPages` in `PageInfo` in case we wanted to add `PageInfo` later somewhere.